mines/interaction_list[] agent_interaction
